### ISSUE TITLE
为什么还要交换再去一次损失呢？，论文没找到这个做法呢

### ISSUE THREAD

**Issue by cqray1990** [lhwcv/mlsd_pytorch#31]
# swap pt
    pred_dis2 = torch.cat((pred_dis[:, 2:, :, :], pred_dis[:, :2, :, :]), dim=1)
    displacement_loss2 = F.smooth_l1_loss(pred_dis2, gt_dis, reduction='none').sum(axis=[1])
    displacement_loss = displacement_loss1.min(displacement_loss2)
